Signal/src/ViewControllers/ConversationView/Cells/OWSBubbleView.m

```diff
@@ -132,8 +134,12 @@ - (void)setBubbleColor:(nullable UIColor *)bubbleColor
 
 - (void)updateLayers
 {
-    OWSAssert(self.maskLayer);
-    OWSAssert(self.shapeLayer);
+    if (!self.maskLayer) {
```

Comment:
Same question - shapeLayer and maskLayer are set in `init` and never unset.

Why do we have the nil checks here and elsewhere?